source/architecture/multipage.md

```diff
@@ -0,0 +1,177 @@
+# Build a multipage site
+
+You can create a technical documentation site that splits its content across multiple pages.
+
+This is suitable for documentation sites that have too much content for the single page format.
+
+You should use the search feature [link] with multipage documentation sites.
+
+## Basic multipage
+
+You can split the content into multiple individual pages. An example is the [GOV.UK PaaS technical documentation](https://docs.cloud.service.gov.uk/).
+
+### Repo folder structure
+
+A typical single page documentation repo has this folder structure:
+
+[IMAGE]
+
+A basic multipage documentation repo can have this structure:
+
+[IMAGE]
+
+You must amend the documentation repo folder structure to reflect this structure.
+
+### Amend the tech-docs.yml file
+
+Add this to the end of your project’s `.../config/tech-docs.yml` file, or set it to true if it is already there:
```

Comment:
We shouldn't assume where the user is in relation to their PATH so we can just say about "amending the `tech-docs.yml` file".

Reply:
agree